tests/python/gaia-ui-tests/gaiatest/apps/email/app.py

```diff
@@ -31,6 +31,7 @@ class Email(Base):
     _search_textbox_locator = (By.CSS_SELECTOR, 'form[role="search"]')
     _email_subject_locator = (By.XPATH, '//a[@data-index!="-1"]/div/span[text()="%s"]')
     _back_button_locator = (By.CLASS_NAME, 'sup-back-btn')
+    header_locator = (By.CSS_SELECTOR, '.msg-list-header')
```

Comment:
We use this attribute outside of the class. I think we should keep it as public.